pyccel/codegen/printing/cwrappercode.py

```diff
@@ -672,6 +678,82 @@ def get_PyBuildValue(self, used_names, variable):
 
         return collect_var, cast_function
 
+    def insert_constant(self, mod_name, var_name, var, collect_var):
+        """
+        Insert a variable into the module
+
+        Parameters
+        ----------
+        mod_name    : str
+                      The name of the module variable
+        var         : Variable
+                      The module variable
+        collect_var : Variable
+                      A PyObject* variable to store temporaries
+        """
+        if var.rank != 0:
+            errors.report("Arrays cannot currently be exposed to Python",
+                    severity='warning', symbol=var)
```

Comment:
I think this warning should be more specific. I suggest "Global arrays (defined at the module level) cannot [...]"

By the way, can we add the exact file location to the warning?

Reply:
File location is a bit trickier, but I have managed to do it via the assign node: 745a440